### PR TITLE
Improved message delivery (status-lib)

### PR DESCRIPTION
**Pull request by alwx**
https://github.com/status-im/status-react/pull/204

It fixes #191

### CODE REVIEW

File: src/cljs/status_im/protocol/api.cljs [hunk 1]
```diff
@@ -104,37 +103,39 @@
...
        (doseq [group-id active-group-ids]
-         (listen connection handle-incoming-whisper-msg {:topic [group-id]}))
+         (listen connection handle-incoming-whisper-message {:topic [group-id]}))
        (doseq [topic topics]
-         (listen connection handle-incoming-whisper-msg {:topic topic}))
+         (listen connection handle-incoming-whisper-message {:topic topic}))
        (do-periodically (* 60 10 1000) send-online)
        (invoke-user-handler :initialized {:identity account})))))
 
+(defn init-pending-messages [pending-messages]
```
Comment (rasom) [inline src/cljs/status_im/protocol/api.cljs:115]: Do we really need this function?

Comment (alwx) [reply src/cljs/status_im/protocol/api.cljs:115]: Of course, yes. This function fills the list of pending messages. If you remove this function everything else will be much more useless.

Comment (rasom) [reply src/cljs/status_im/protocol/api.cljs:115]: I just mean that we could use `set-pending-messages` directly. :)

Comment (rasom) [reply src/cljs/status_im/protocol/api.cljs:115]: Oh I see, that's definition of API, sorry, I missed that point.

**Pull request closed without merging**